closed/autoconf/custom-hook.m4

```diff
@@ -441,36 +441,33 @@ AC_DEFUN([OPENJ9_CHECK_NASM_VERSION],
   OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU($host_cpu)
 
   if test "x$OPENJ9_CPU" = xx86-64 ; then
-    AC_PATH_PROG([NASM],[nasm])
-    if test "x$NASM" != x ; then
-      AC_MSG_CHECKING([whether nasm version requirement is met])
+    BASIC_REQUIRE_PROGS(NASM , nasm)
```

Comment:
In jdknext, the macro was renamed from `BASIC_REQUIRE_PROGS` to `UTIL_REQUIRE_PROGS`.
Also, please continue the convention in this file of using quotes (`[]`).